src/inc/quic_tls.h

```diff
@@ -230,6 +250,41 @@ typedef struct QUIC_TLS_PROCESS_STATE {
 
 } QUIC_TLS_PROCESS_STATE;
 
+#define QUIC_TLS_RESUMPTION_TICKET_VERSION 1
+
+//
+// In-memory representation of a resumption ticket.
+//
+typedef struct QUIC_TLS_RESUMPTION_TICKET {
+    //
+    // Version of this structure.
+    //
+    uint64_t TicketVersion;
```

Comment:
We don't really need an 8 byte version do we? Same for all the length fields below. I believe `uint16_t` should be fine for the lengths. Even a 1 byte version is probably plenty. We can always interpret version of 0xFF to have an additional subversion field in the future....

Reply:
Well, these were all supposed to be `QUIC_VAR_INT` so that QuicVarIntDecode could use them without issue.
Honestly, I'm not convinced I even need this structure anymore.  What do you think?

Reply:
Just do whatever is simplest.